docs/_documentations/mdt-vsc-installinfo.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: docs
+title: Installing Codewind for VS Code
+description: Installing Codewind for VS Code
+keywords: install, installing, add, adding, tools, VS Code, Eclipse, installing Codewind for VS Code
+duration: 1 minute
+permalink: mdt-vsc-installinfo
+type: document
+order: 
+parent: 
+---
+
+# Installing Codewind for VS Code
+
+The Codewind installation includes two parts:
+1. The VS Code extension installs when you install Codewind from the VS Code Marketplace or when you install by searching in the **VS Code Extensions** view.
+2. The Codewind back end containers install after you click **Install** when you are prompted. After you click **Install**, the necessary images are downloaded from the internet. The download is approximately 1 GB.
+
+The following images are pulled. These images together form the Codewind back end:
+- `eclipse/codewind-initialize-amd64`
+- `eclipse/codewind-performance-amd64`
+- `eclipse/codewind-pfe-amd64`
+
+## Using Codewind
+When the installation is complete, the extension is ready to use, and you are prompted to open the Codewind workspace. For more information about using Codewind, see [Tutorial: Codewind for VS Code](mdt-vsc-tutorial.html).
```

Comment:
@sishida and @tetchel : Everything looks good to me. In terms of the link, do we have any documentation explaining workspaces? If we do then the section about workspaces opening should have that link (e.g. To learn more about workspace go here).

Reply:
@thisguy-1: We have this file, https://www.eclipse.org/codewind/mdt-vsc-getting-started.html, that states, "Codewind creates a folder called codewind-workspace within your home directory (C:\codewind-workspace on Windows) to contain your projects. When the installation is complete, you can open the codewind-workspace folder or a project within the workspace as your VS Code workspace. The tools offer to open the workspace for you if it’s not open already."

What do you think of removing this link in line 25 and maybe just copy/pasting this workspace info instead? (I'm not a fan of copy/paste from one doc to another, but it might be clearer to the user to have the info right there.)

Reply:
I think that works since it seems there isn't a specific doc reg. the workspaces. So, copy and paste this info, but keep the link so that they can learn next steps anyway.